server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/DomainQuotaRoutesTest.java

```diff
@@ -447,6 +448,54 @@ void putQuotaShouldUpdateBothQuota() {
         assertThat(maxQuotaManager.getDomainMaxStorage(TROUVÉ_COM)).contains(QuotaSize.size(42));
     }
 
+    @Test
+    void putQuotaWithNegativeCountShouldFail() throws MailboxException {
```

Comment:
don't you miss `putQuotaWithNegativeOneShouldSucess`? and idem for other tests

Reply:
Those tests already existed : 
`putCountShouldSetToInfiniteWhenMinusOne`
and
`putSizeShouldSetToInfiniteWhenMinusOne`